10-angular/custom_directives/readme.md

```diff
@@ -740,112 +732,10 @@ A "soft" rule or guideline for when to use directives or controllers is:
 - `template` and `templateUrl`
 - `link`
 
-##### What's the *right* way to use them?
-
-There isn't one.
-
-### The story of Robin and Angular
```

Comment:
I miss this, please include some kind of throwback reference to Robin's real talk on angular.